MaterialLib/Fluid/Viscosity/PressureDependentViscosity.h

```diff
@@ -0,0 +1,64 @@
+/*!
+   \file  PressureDependentViscosity.h
+   \brief Declaration of class for the pressure dependent viscosity model.
+
+   \author Wenqing Wang
+   \date Jan 2015
+
+   \copyright
+    Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+               Distributed under a Modified BSD License.
+               See accompanying file LICENSE.txt or
+               http://www.opengeosys.org/project/license
+*/
+#ifndef PRESSURE_DEPENDENT_VISCOSITY_H_
+#define PRESSURE_DEPENDENT_VISCOSITY_H_
+
+#include <string>
+#include <algorithm>    // std::max
+
+#include "ViscosityType.h"
+
+namespace MaterialLib
+{
+
+/// Pressure dependent viscosity model,
+class PressureDependentViscosity
```

Comment:
I think the most of models are named already. My idea is to create a class for each mode, which may be function of p, T or other variables,  and  name the class after the name of the model.